test/e2e/far_e2e_test.go

```diff
@@ -163,8 +126,18 @@ var _ = Describe("FAR E2e", func() {
 				remediationTimes = append(remediationTimes, time.Since(startTime))
 			})
 		})
+	}
+
+	Context("stress cluster with OutOfServiceTaint remediation strategy", func() {
```

Comment:
The context doesn't match the RemediationStrategy.

```suggestion
	Context("stress cluster with ResourceDeletion remediation strategy", func() {
```

Reply:
Sorry. Fixed it.